src/state/data-models/types.js

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line
+export const SET_DATA_MODELS = 'datamodels/set-data-models';
+export const SET_DATA_MODELS_PAGED = 'datamodels/set-data-models-paged';
```

Comment:
No need for an extra action. Use SET_DATA_MODELS, and use selectors to handle client-side paging